specification/compute/resource-manager/readme.md

```diff
@@ -206,6 +206,21 @@ directive:
       - ACS service is deprecated so a PATCH endpoint won't be implemented
 
 ```
+
+### Tag: package-2019-11-01-Disks
```

Comment:
Why we append a disk suffix to this tag? To me, this tag works as a fully functional compute service